docs/UsingNavigators.md

```diff
@@ -78,7 +78,7 @@ render() {
     <Navigator
       initialRoute={{ title: 'My Initial Scene', index: 0 }}
       renderScene={(route, navigator) => {
-        <MyScene title={route.title} />
+        return <MyScene title={route.title} />
```

Comment:
Here we can remove the curly braces instead, since it is a single line component.